### DISCUSSION TITLE
Vercel로 React 프로젝트 배포해보는 방법

### DISCUSSION THREAD

**Discussion by hjlee2778** [prgrms-fe-devcourse/NFE1_2_2_MUX#121] [D. 아이디어&레퍼런스]
저희가 시간이 남는다면, Vercel로 배포까지 해보면 좋을 듯 해요!!
다른 분들도 이용할  수 있게 해서 노래 추천 받고 싶슴니다!

배포하기!
1. https://vercel.com/ 👈우선 vercel 사이트에 접속해서 회원가입을 해준다.

2. start deploying을 누르면

3. Import Git Repository를 할 수 있는 페이지가 나온다.

4. github을 사용하기 때문에 Continue with GitHub을 누르고 Import Git Repository에서 github에 있는 내 repository에서 import해 올 repo를 선택한다.

5. import해 온 repo에서 import 버튼을 누르면 Configure Project라고 deploy하기위한 설정을 해주는 페이지가 나온다.

가장 간단한 방법인데, 이런식으로 금방 할 수 있긴 하더라구요!
한번 해보는 것도 좋을 것 같습니다!

**Comment by ryan-g00** [comment]
좋습니다 화이팅!
